datastore-benchmarks/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "benchmarks",
+  "version": "1.0.0",
+  "private": true,
+  "description": "CRDT benchmarks",
+  "scripts": {
+    "clean": "rm -rf dist",
+    "dist": "npm run clean; rollup -c; gzip --keep dist/*",
+    "start": "npm run dist && node dist/benchmark.js",
+    "watch": "npm run dist && rollup -wc",
+    "lint": "standard && tsc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dmonad/crdt-benchmarks.git"
+  },
+  "keywords": [
+    "CRDT",
+    "benchmark"
+  ],
+  "author": "Kevin Jahns",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/dmonad/crdt-benchmarks/issues"
+  },
+  "homepage": "https://github.com/dmonad/crdt-benchmarks#readme",
+  "dependencies": {
+    "yjs": "13.0.0-102",
```

Comment:
Could you please use a current version of Yjs ^^ ? This version is almost a year old. Yjs is currently 13.3.2 and is now 10x faster using new encoding techniques.